### PR TITLE
Added some stories

### PR DESCRIPTION
**Pull request by HendrikBK**
Added the following stories to the book:
The Lost Race
The Dream Snake
The Hyena
The Shadow Kingdom
The Mirrors of Tuzun Thune
Rattle of Bones
Skull-Face
Skulls in the Stars
The Haunter of the Ring
The Grisly Horror
The Fire of Ashurbanipal
Black Hound of Death
Black Canaan
Dig Me no Grave

### CODE REVIEW

File: src/epub/css/local.css [hunk 1]
```diff
@@ -65,6 +68,22 @@ p[epub|type~="se:letter.dateline"]{
...
 
+#the-grisly-horror [epub|type~="z3998:letter"] header{
+	text-align: left;
+}
+
+[epub|type~="z3998:letter"] header{
+	text-align: right;
+}
+
+[epub|type~="z3998:letter"] header + p{
```
Comment (acabal) [inline src/epub/css/local.css]: This has no effect, you can remove it

**Review by acabal — CHANGES_REQUESTED** [review]
Thanks Hendrik. There's a note about CSS and also, all ordinals must be Roman numerals. Can you please convert all ordinals in this PR and add the `z3998:roman` semantic? Then the ToC will have to be updated.

File: src/epub/css/local.css (hunk 1 above)
Comment (acabal) [reply src/epub/css/local.css]: `[epub|type~="z3998:letter"] header + p`

**Comment by acabal** [comment]
Also, something is broken in one of the filenames. `se lint` fails with a file not found error. Please fix that and make sure `se build --check-only` succeeds too.

**Comment by HendrikBK** [comment]
I think I fixed all issues. The issue with the file name was that I modernized _Ashurbanipal_ but forgot to change the file-name.

**Comment by acabal** [comment]
OK, looks good, thanks!

**Pull request merged**